WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1123,9 +1124,8 @@ public void onClick(DialogInterface dialog, int which) {
                         String entry = input.getText().toString();
                         if (!TextUtils.isEmpty(entry) && !mEditingList.contains(entry)) {
                             mEditingList.add(entry);
-                            list.setAdapter(new ArrayAdapter<>(getActivity(),
-                                    R.layout.wp_simple_list_item_1,
-                                    mEditingList));
+                            mAdapter = null;
+                            list.setAdapter(getAdapter());
```

Comment:
I think that with `RecyclerView`, instead of resetting adapter every time the new word is added, we can just call  ` getAdapter().notifyItemInserted(getAdapter().getItemCount() - 1);`, it's already tracking data from `mEditingList` so we just need to let adapter know that there is a new item.

And maybe we can add scrolling to the added item after this call, with something like this:
```
list.post(new Runnable() {
                                @Override
                                public void run() {
                                    list.smoothScrollToPosition(getAdapter().getItemCount() - 1);
                                }
                            });
```